src/policy/concrete.rs

```diff
@@ -918,5 +939,12 @@ fn with_huffman_tree<Pk: MiniscriptKey>(
         .pop()
         .expect("huffman tree algorithm is broken")
         .1;
-    Ok(node)
+
+#[cfg(feature = "compiler")]
+fn combination(n: usize, k: usize) -> usize {
```

Comment:
Whenever we call this, we need to make sure that we are having an overflow.

Reply:
This function isn't supposed to be there. Seems I messed up while squashing the commits, my bad!